application/src/main/java/net/furizon/backend/infrastructure/pretix/service/CachedPretixInformation.java

```diff
@@ -23,23 +32,35 @@
 @RequiredArgsConstructor
 @Slf4j
 public class CachedPretixInformation implements PretixInformation {
-    @NotNull
-    private final UseCaseExecutor useCaseExecutor;
+    @NotNull private final UseCaseExecutor useCaseExecutor;
 
-    @NotNull
-    private final AtomicReference<Event> currentEvent = new AtomicReference<>(null);
+    @NotNull private final AtomicReference<Event> currentEvent = new AtomicReference<>(null);
 
-    @NotNull
-    private final AtomicReference<Integer> questionSecretId = new AtomicReference<>(-1);
+    //Event Struct
 
-    @NotNull
-    private final Cache<Integer, QuestionType> questionTypeIdsCache = Caffeine.newBuilder().build();
+    //Contains tickets, memberships, sponsors, rooms
+    @NotNull private final Cache<CacheItemTypes, Set<Integer>> itemIdsCache = Caffeine.newBuilder().build();
 
-    @NotNull
-    private final Cache<Integer, String> questionIdentifiersCache = Caffeine.newBuilder().build();
+    //Questions
+    @NotNull private final AtomicReference<Integer> questionSecretId = new AtomicReference<>(-1);
+    @NotNull private final Cache<Integer, QuestionType> questionIdToType = Caffeine.newBuilder().build();
+    @NotNull private final Cache<Integer, String> questionIdToIdentifier = Caffeine.newBuilder().build();
+    @NotNull private final Cache<String, Integer> questionIdentifierToId = Caffeine.newBuilder().build();
```

Comment:
per line for annotation (I know it more about personal preferences, but is more standard way for java )

Reply:
But it looks better (for me) :((((((((((

Reply:
well, if you like this way let's do it this way, can you find annotation rule for checkstyle so we can have that style?

Reply:
There should be already! Or, at least, it looks like checkstyle is not angry at all about this hahah